src/main/webapp/i18n/de/course.json

```diff
@@ -96,7 +96,14 @@
                 "title": "Kursregistrierung für Studierende aktiviert",
                 "description": "Aktiviere diese Checkbox, damit sich Studierende selbständig im Kurs registrieren können, sofern dieser aktiv ist (zwischen Start und Ende des Kurses). Du kannst optional eine Nachricht eingeben, die Studierende bei der Registrierung bestätigen müssen."
             },
+            "enrollmentStartDate": "Einschreibungs Start",
+            "enrollmentEndDate": "Einschreibungs Ende",
             "registrationConfirmationMessage": "Nachricht zur Bestätigung der Registrierung",
+            "unenrollmentEnabled": {
+                "title": "Kursauschreibung für Studierende aktiviert",
```

Comment:
"Ausschreibung" is a bit confusing vor me, since it could also be something like an advertisement.
But I cannot find any better translation, maybe someone else can think of one. Otherwise leave it as is

Reply:
I agree but the only better alternative I can think of would be "Abmeldung" which conflicts with the new convention.

Reply:
The Stud.IP LMS (alternative to Moodle) uses ‘eintragen’ and ‘austragen’ for course (un-)enrolment. That might be a good alternative.

Reply:
I changed it to `austragen` as @b-fein suggested.